packages/adapter-mongodb/src/index.ts

```diff
@@ -82,8 +82,7 @@ export const format = {
   },
 }
 
-/** Converts from string to ObjectId */
-export function _id(hex?: string) {
+function _id(hex?: string) {
```

Comment:
Removing these exports, as I think, is not used for our users? ( _seems more like an internal function_ ). 

If exported, then it ends in the middle of the documentation, looking like something random:

<img width="800" alt="image" src="https://user-images.githubusercontent.com/5938217/224483902-0e186068-79fc-417c-8eb1-52e1a52f3466.png">